src/config.ts

```diff
@@ -0,0 +1,107 @@
+
+
+export class Configuration {
+
+    /**
+    * Browser-specific constant to adjust CanvasRenderingContext2D.shadowBlur value,
+    * which is unitless and not rendered equally across browsers.
+    *
+    * Values that work quite well (as of October 2017) are:
+    * - Chrome: 1.5
+    * - Edge: 1.75
+    * - Firefox: 0.9
+    * - Safari: 0.95
+    *
+    * @since 2.0.0
+    * @type Number
+    * @default 1
+    */
+    browserShadowBlurConstant = 1
+
+
+    /**
+     * Pixel per Inch as a default value set to 96. Can be changed for more realistic conversion.
+     */
+    DPI = 96
+
+    /**
+     * Pixel limit for cache canvases. 1Mpx , 4Mpx should be fine.
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    perfLimitSizeTotal = 2097152
+
+    /**
+     * Pixel limit for cache canvases width or height. IE fixes the maximum at 5000
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    maxCacheSideLimit = 4096
+
+    /**
+     * Lowest pixel limit for cache canvases, set at 256PX
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    minCacheSideLimit = 256
+
+    /**
+     * if webgl is enabled and available, textureSize will determine the size
+     * of the canvas backend
+     * @since 2.0.0
+     * @type Number
+     * @default
+     */
+    textureSize = 2048
+
+    /**
+     * When 'true', style information is not retained when copy/pasting text, making
+     * pasted text use destination style.
+     * Defaults to 'false'.
+     * @type Boolean
+     * @default
+     * @deprecated
+     */
+    disableStyleCopyPaste = false
+
+    /**
+     * Enable webgl for filtering picture is available
+     * A filtering backend will be initialized, this will both take memory and
+     * time since a default 2048x2048 canvas will be created for the gl context
+     * @since 2.0.0
+     * @type Boolean
+     * @default
+     */
+    enableGLFiltering = true
+
+
+    /**
+     * If disabled boundsOfCurveCache is not used. For apps that make heavy usage of pencil drawing probably disabling it is better
+     * @default true
+     */
+    cachesBoundsOfCurve = true
+
+    /**
+     * Skip performance testing of setupGLContext and force the use of putImageData that seems to be the one that works best on
+     * Chrome + old hardware. if your users are experiencing empty images after filtering you may try to force this to true
+     * this has to be set before instantiating the filtering backend ( before filtering the first image )
+     * @type Boolean
+     * @default false
+     */
+    forceGLPutImageData = false
+
+    NUM_FRACTION_DIGITS = 2
```

Comment:
We should leave a note here, that this represent the number of digits when exporting to json. Is a good idea to bump this to 4 or more for designs app, and that is kept to 2 otherwise testing the library is a mess

Reply:
I think we should remove this strange const and introduce it in the options object passed to `toObject` and `toSVG`
It is really bad IMO, e.g. #8178

Reply:
And fix tests accordingly

Reply:
bumped to 4